### PR TITLE
testing/mbuffer: initial package commit based on version 20171011

### PR DESCRIPTION
**Pull request by Fusl**

### CODE REVIEW

**Comment by kaniini** [comment]
please flatten

**Comment by Fusl** [comment]
@kaniini done

File: testing/mbuffer/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributor: Katie Holly <holly@fuslvz.ws>
+# Maintainer: Katie Holly <holly@fuslvz.ws>
+pkgname=mbuffer
+pkgver=20171011
+pkgrel=0
+pkgdesc="mbuffer buffers I/O operations and displays the throughput rate"
+url="http://www.maier-komor.de/mbuffer.html"
+arch="all"
+license="GPL-3.0"
```
Comment (ncopa) [inline testing/mbuffer/APKBUILD]: We are moving towards [SPDX](https://spdx.org/licenses/) so it should be `license="GPL-3.0-only"`.

File: testing/mbuffer/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+depends=""
```
Comment (ncopa) [inline testing/mbuffer/APKBUILD]: This row could be removed (but it does not really hurt to keep it either)

File: testing/mbuffer/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+makedepends="linux-headers libressl-dev"
+install=""
```
Comment (ncopa) [inline testing/mbuffer/APKBUILD]: This could be removed too.

File: testing/mbuffer/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+subpackages="$pkgname-doc"
+source="
+	http://www.maier-komor.de/software/mbuffer/mbuffer-$pkgver.tgz
+	"
```
Comment (ncopa) [inline testing/mbuffer/APKBUILD]: When there is only a single url we normally have it on a single line:
`source="http://www.maier-komor.de/software/mbuffer/mbuffer-$pkgver.tgz"`

File: testing/mbuffer/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+builddir="$srcdir"/mbuffer-$pkgver
+
+prepare() {
+	cd "$builddir"
+	default_prepare
+}
```
Comment (ncopa) [inline testing/mbuffer/APKBUILD]: The entire `prepare()` function can be removed since the default is to run `default_prepare`

**Pull request closed without merging**